test/JSON/stringifyobjects.js

```diff
@@ -0,0 +1,20 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+if (this.WScript && this.WScript.LoadScriptFile) { // Check for running in ch
+    this.WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+}
+
+const tests = [
+    {
+        name: "Stringify error",
+        body: function () {
+            const err = new Error("message");
+            assert.areEqual('{"description":"message","message":"message"}', JSON.stringify(err));

```

Comment:
does it make sense to call stringify on every object chakra can produce?

Reply:
Possibly? I took a quick look through all of our JSON tests to try to find a place to put this one, and it looked like every other JSON test we have JSON.parse()s a string and then JSON.stringify()s it to make sure the strings look similar. One possible downside of adding exhaustive tests here are because they depend on the order of enumeration, if we ever change that then we would have go go through a ton of tests to update it as well. This is a problem we have with a lot of node-chakracore tests right now, where many tests they add fail because they depend on V8's current enumeration ordering for output.